src/main/scala/BintrayKeys.scala

```diff
@@ -32,6 +32,9 @@ trait BintrayKeys {
   val bintrayCredentialsFile = settingKey[File](
     "File containing bintray api credentials")
 
+  val bintrayCredentials = settingKey[Option[BintrayCredentials]](
+    "Bintray api credentials")
```

Comment:
sbt already has a setting for managing credentials - the `credentials` setting.  We shouldn't be introducing a new setting here when sbt already provides this, it's just frustrating because it means bintray is doing something in a different way to every other plugin that does credentials out there, and also means you usually need to define the bintray credentials twice, once for the bintray resolver to use via the standard mechanism, and the other for the bintray plugin to use via this setting.

The other advantage of using `credentials` is that you can add these credentials globally in `~/.sbt/0.13/global.sbt`, without having to add the bintray plugin globally.